src/components/hosts/hardwareInfo.ts

```diff
@@ -64,7 +64,7 @@ export const getHostRowHardwareInfo = (inventory: Inventory): HostRowHardwareInf
   }
 
   return {
-    serialNumber: inventory.systemVendor?.serialNumber || DASH,
+    serialNumber: inventory.systemVendor?.serialNumber || 'Virtual machine',
```

Comment:
```suggestion
    serialNumber: inventory.systemVendor?.serialNumber || 'None (virtual machine)',
```

This might flow a little better: "What's the serial number? Oh, it's `None`, because it's a virtual machine"

Also, is not having a serial number a reliable way of detecting a virtual machine? Could there be hosts that we just can't detect a serial number for, but they're actually bare metal and not VM-based? 🤷‍♂️

Reply:
@ori-amizur Can you please comment ^^^ ?